packages/core/src/L3/nodejs/scripts/CleanScriptPath.ts

```diff
@@ -0,0 +1,11 @@
+import { Construct } from "constructs";
+import { CleanScript } from "./CleanScript";
+
+export class CleanScriptPath extends Construct {
+  constructor(scope: Construct, id: string, path: string | string[]) {
+    super(scope, id);
+
+    let paths: string[] = typeof path === "string" ? [path] : path;
+    CleanScript.of(this).addPaths(paths);
+  }
+}
```

Comment:
```suggestion
export class CleanScriptPath extends Construct {
  readonly path: string

  constructor(scope: Construct, id: string, path: string) {
    super(scope, id);

    this.path = path
  }
}
```

I think this is correct, otherwise the class is kinda pointless?

Reply:
Yeah you could run `CleanScript.of(this).addPaths(paths);` but `CleanScriptPath` is an abstraction around that, it works similar with other constructs in the project.